### ISSUE TITLE
Ajour won't respect release.json when installing from GitHub

### ISSUE THREAD

**Issue by Silun** [ajour/ajour#686]
**Describe the bug**
When installing from GitHub, Ajour may install the wrong version, e.g. of this repo:

https://github.com/raethkcj/RatingBuster

It has a release file that should tell Ajour what to install, but it still installs the wrong version by default for Classic TBC at least.

This is the content of the `release.json`:

```json
{
    "releases": [{
        "filename": "RatingBuster-v1.0.12-bcc.zip",
        "nolib": false,
        "metadata": [{
            "flavor": "bcc",
            "interface": 20501
        }]
    }, {
        "filename": "RatingBuster-v1.0.12-classic.zip",
        "nolib": false,
        "metadata": [{
            "flavor": "classic",
            "interface": 11307
        }]
    }]
}
```

**To Reproduce**
1. Install Ratingbuster for TBC CLassic via Ajour from `https://github.com/raethkcj/RatingBuster`
2. Check the version it has installed in the toc file
3. See that it has installed version 11307

**Expected behavior**
It should have installed the version indicated for 20501

**Software involved**
Please complete the following information:
 - OS: Windows 10
 - Ajour version: 1.2.2
 - TBC Classic

**Additional context**

**Log Output**
Log of updating the addon today, same issue applies.
[ajour.log](https://github.com/ajour/ajour/files/6816657/ajour.log)

**Comment by casperstorm** [comment]
I'll fix this bug tonight. I can reproduce it and know what the problem is.
Sorry for the troubles.

**Comment by casperstorm** [comment]
Alright, the bug, and also proper support for `release.json` has now been added #687.
It will be release once reviewed.

**Comment by Silun** [comment]
Awesome, thank you!

**Comment by casperstorm** [comment]
Released in 1.2.3-rc.1.
Make sure to set Ajour Update Channel to Beta in order download this pre-release.
